app/gui/src/project-view/components/widgets/FileBrowserWidget.vue

```diff
@@ -291,27 +308,31 @@ onMounted(() => {
     <div v-else :key="currentDirectory?.id ?? 'root'" class="listing contents">
       <TransitionGroup>
         <FileBrowserEntry
-          v-if="editedAsset?.asset === newDirPlaceholder"
+          v-if="editedAsset && editedAsset.asset == null"
           :key="keyOverride.get(newDirPlaceholder) ?? newDirPlaceholder"
           icon="folder"
           :title="editedAsset.name"
           :editingState="editedAsset.state"
-          @nameAccepted="acceptName($event, 'create folder')"
+          :renamable="false"
+          @nameAccepted="acceptName($event)"
         />
         <FileBrowserEntry
           v-for="entry in directories"
           :key="keyOverride.get(entry.id) ?? entry.id"
           icon="folder"
-          :title="editedAsset?.asset === entry ? editedAsset.name : entry.title"
-          :editingState="editedAsset?.asset === entry ? editedAsset.state : undefined"
+          :title="editedAsset?.asset?.id === entry.id ? editedAsset.name : entry.title"
+          :editingState="editedAsset?.asset?.id === entry.id ? editedAsset.state : undefined"
+          :renamable="editedAsset == null"
```

Comment:
The `FileBrowserEntry` doesn't need this value eagerly--it would be better for scalability to avoid changing a prop on all directory components (causing Vue to update them) when this value changes.